README.md

```diff
@@ -1,6 +1,6 @@
 # Parse.ly
 
-Stable tag: 3.3.0  
+Stable tag: 3.3.1
```

Comment:
```suggestion
Stable tag: 3.3.1  
```

Reply:
Actually, this was on purpose. Is there a reason to keep the whitespace?

Reply:
Yes, the lines below also do have it. It results in a line break. Else, the lines appear joined together.

Reply:
I do find a bad idea to use it though, as it is invisible and can be removed by mistake. I would prefer using `\` for this, but don't know why double space was used in the first place, and if replacing it could have side effects.

Reply:
I would be very careful with this, since it is also read by WordPress / WordPress.org.

Reply:
Yes, this is why I haven't touched it 😃